stable/kube-bench/templates/cron.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: {{ template "cronjob.apiVersion" . }}
+kind: CronJob
+metadata:
+  name: kube-bench
+spec:
+  schedule: {{ quote .Values.cronjob.schedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          hostPID: true
+          restartPolicy: Never
+          containers:
+            - name: kube-bench
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              command: {{ template "cronjob.command" . }}
+              resources:
+{{ toYaml .Values.resources | indent 14 }}
+              volumeMounts:
+                - name: var-lib-kubelet
+                  mountPath: /var/lib/kubelet
+                  readOnly: true
+                - name: etc-systemd
+                  mountPath: /etc/systemd
+                  readOnly: true
+                - name: etc-kubernetes
+                  mountPath: /etc/kubernetes
+                  readOnly: true
+          volumes:
+            - name: var-lib-kubelet
+              hostPath:
+                path: "/var/lib/kubelet"
+            - name: etc-systemd
+              hostPath:
+                path: "/etc/systemd"
+            - name: etc-kubernetes
+              hostPath:
+                path: "/etc/kubernetes"
```

Comment:
It's probably nicer to include the standard options that allow the user to customise the scheduling for their env. e.g. https://github.com/deliveryhero/helm-charts/blob/master/stable/k8s-event-logger/templates/deployment.yaml#L41-L52